packages/js/product-editor/src/blocks/shipping-dimensions/block.json

```diff
@@ -0,0 +1,25 @@
+{
+	"$schema": "https://schemas.wp.org/trunk/block.json",
+	"apiVersion": 2,
+	"name": "woocommerce/product-shipping-dimensions-fields",
+	"title": "Product shipping dimensions fields",
+	"category": "woocommerce",
+	"description": "The product shipping dimensions fields.",
+	"keywords": [ "products", "shipping", "dimensions" ],
+	"textdomain": "default",
+	"attributes": {
+		"name": {
```

Comment:
Doesn't really matter too much since I know this is not actually being used, but I wonder if there's a name we can use to make this obvious this is not an actual attribute being used.  E.g., `__content` or `contentEditable`.

Not great name suggestions and not something we have to do, just thinking out loud.

Reply:
Yes, It's true, since that prop is symbolic there, it makes sense to have something like you mentioned.

Reply:
done -> https://github.com/woocommerce/woocommerce/pull/37683/commits/c74cef75de43fdb04a4b75bccc53e22d6cea08cc